packages/web-app-search/src/composables/index.ts

```diff
@@ -0,0 +1,3 @@
+export const useSearchActive = (): boolean => {
```

Comment:
Since you put this in the `composables` folder I'd expect it to return reactive data, i.e. return type being `Ref<Boolean>`, not `boolean`. Since you don't need the data in a reactive way I'd recommend renaming this to `isSearchActive` and moving it into some helper package. The way it's implemented is not a composable, yet.